ramls/request.json

```diff
@@ -221,6 +221,30 @@
       "description": "Tags",
       "$ref": "raml-util/schemas/tags.schema"
     },
+    "printDetails": {
```

Comment:
As you are adding a new section in request.json I think you need to bump up the interface version of this module if not done already.

Reply:
fixed-> https://github.com/folio-org/mod-circulation-storage/pull/487/commits/a0e8d4b79d11d141dc34c837f1df0a36a225f6c8